test/functional/interface_zmq.py

```diff
@@ -511,16 +513,16 @@ def test_mempool_sync(self):
                     # Detected "R" gap, means this a conflict eviction, and mempool tx are being evicted before its
                     # position in the incoming block message "C"
                     if label == "R":
-                        assert mempool_sequence > expected_sequence
+                        assert_greater_than(mempool_sequence, expected_sequence)
                         r_gap += mempool_sequence - expected_sequence
                     else:
                         raise Exception(f"WARNING: txhash has unexpected mempool sequence value: {mempool_sequence} vs expected {expected_sequence}")
             if label == "A":
-                assert hash_str not in mempool_view
+                assert_true(hash_str not in mempool_view)
                 mempool_view.add(hash_str)
                 expected_sequence = mempool_sequence + 1
             elif label == "R":
-                assert hash_str in mempool_view
+                assert_true(hash_str in mempool_view)
```

Comment:
what is the motivation to change this?

Reply:
The only reason is to maintain consistency between error code format and code style. I usually try to avoid mixing different types of assert (from library and native assert) in the tests

Reply:
It is fine if you do this personally in your own projects, but for bitcoin-core this is impossible to maintain. Someone is going to break the consistency at some point. We can't handle this if there are floods of follow-up pull requests. Also, if there is a linter to make lib assert illegal, it puts an unnecessary burden on developers for style reasons.

Reply:
Mh I agree with the difficulty to add a change of style at this point on Bitcoin core, I only propose one way here and I'm super happy to revert the change and to have this great exchange of idea